common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1517,6 +1517,11 @@ public enum PUBLIC_SHARE_VISIBILITY { samePrimaryDomain, all, none };
     // Storage Management
     public final static KnownKey zimbra_sm_blob_mover_parallelism_level = KnownKey.newKey(5);
     public final static KnownKey zimbra_s3_connection_request_timeout_ms = KnownKey.newKey(20000);
+
+    // ZBUG-2547
```

Comment:
Can you please add some details here as well?

Reply:
done